CatProducer/python/producers/jetProducer_cfi.py

```diff
@@ -3,7 +3,7 @@
 catJets = cms.EDProducer('CATJetProducer',
     src = cms.InputTag('slimmedJets'),
     rho = cms.InputTag('fixedGridRhoFastjetAll'),
-    btagNames = cms.vstring('pfCombinedInclusiveSecondaryVertexV2BJetTags','pfCombinedMVAV2BJetTags',"inclusiveCandidateSecondaryVerticesCvsL","pfCombinedCvsLJetTags","pfCombinedCvsBJetTags"),
+    btagNames = cms.vstring('pfCombinedInclusiveSecondaryVertexV2BJetTags',"pfDeepCSVDiscriminatorsJetTags:BvsAll","pfDeepCSVJetTags:probb","pfDeepCSVJetTags:probbb","pfDeepCSVDiscriminatorsJetTags:CvsB","pfDeepCSVDiscriminatorsJetTags:CvsL","pfCombinedCvsLJetTags","pfCombinedCvsBJetTags"),
```

Comment:
I think "pfDeepCSVDiscriminatorsJetTags:BvsAll" will not work at all. the right place to add them is "flavTagLabels".

Reply:
Those deep csv taggers work as CSVv2, so this works fine. Note that "pfDeepCSVJetTags:probb"+"pfDeepCSVJetTags:probbb"="pfDeepCSVDiscriminatorsJetTags:BvsAll", for Data.